tests/modules/individuals/resources/test_merge.py

```diff
@@ -0,0 +1,270 @@
+        individual2_id = str(encounter2.individual_guid)
+
+        # this tests as researcher_2, which should trigger a merge-request (owns just 1 encounter)
+        # NOTE: merge_request not yet implmented, so fails accordingly (code 500)
+        data_in = [individual2_id]
+        response = test_utils.post_via_flask(
+            flask_app_client,
+            researcher_2,
+            scopes=('individuals:write',),
+            path=f'/api/v1/individuals/{individual1_id}/merge',
+            data=data_in,
+            expected_status_code=500,
+            response_200={'merged'},
+        )
+        assert response.json['merge_request']
+        assert response.json['message'] == 'Merge failed'
+        assert response.json['blocking_encounters'] == [str(encounter1.guid)]
+
+        # a user who owns none (403 fail, no go)
+        response = test_utils.post_via_flask(
+            flask_app_client,
+            contributor_1,
+            scopes=('individuals:write',),
+            path=f'/api/v1/individuals/{individual1_id}/merge',
+            data=data_in,
+            expected_status_code=403,
+            response_200={'merged'},
+        )
+
+        # anonymous (401)
+        response = test_utils.post_via_flask(
+            flask_app_client,
+            None,
+            scopes=('individuals:write',),
+            path=f'/api/v1/individuals/{individual1_id}/merge',
+            data=data_in,
+            expected_status_code=401,
+            response_200={'merged'},
+        )
+
+    finally:
+        individual_utils.delete_individual(flask_app_client, researcher_1, individual1_id)
+        individual_utils.delete_individual(flask_app_client, researcher_2, individual2_id)
+        sighting1.delete_cascade()
+        sighting2.delete_cascade()
+
+
+@pytest.mark.skipif(
+    test_utils.module_unavailable(
+        'individuals', 'encounters', 'sightings', 'social_groups'
+    ),
+    reason='Individuals module disabled',
+)
+def test_merge_social_groups(db, flask_app_client, researcher_1):
+    from app.modules.social_groups.models import SocialGroup
+
+    individual1_id = None
+    sighting1 = None
+    encounter1 = None
+    individual2_id = None
+    sighting2 = None
+    encounter2 = None
+    try:
+        sighting1, encounter1 = prep_data(db, flask_app_client, researcher_1)
+        individual1_id = str(encounter1.individual_guid)
+        sighting2, encounter2 = prep_data(db, flask_app_client, researcher_1)
+        individual2_id = str(encounter2.individual_guid)
+
+        individual1 = Individual.query.get(individual1_id)
+        individual2 = Individual.query.get(individual2_id)
+
+        # this tests target individual is not in social group
+        groupA_name = 'groupA'
+        groupA_role_from_2 = 'doomed-to-be-merged'
+        members = {individual2_id: {'roles': [groupA_role_from_2]}}
+        social_groupA = SocialGroup(members, groupA_name)
```

Comment:
In this way, your test is using roles that are not defined and means that if you run any of the social group tests first, they may fail. 
You'd be better off using the sociall group utils 
https://github.com/WildMeOrg/houston/blob/4b901e10371024a31abc3cea7d8d42d498e40697/tests/modules/social_groups/resources/utils.py#L117
Then testing that if individual 2 was the matriarch then individual 1 becomes it.

Reply:
ok cool.  using social group utils now.